forum/communications/channelsmiddleware.py

```diff
@@ -0,0 +1,46 @@
+from channels.db import database_sync_to_async
+from channels.middleware import BaseMiddleware
+from rest_framework.exceptions import APIException
+from rest_framework_simplejwt.exceptions import AuthenticationFailed
+from rest_framework_simplejwt.tokens import AccessToken
+from rest_framework_simplejwt.exceptions import TokenError, InvalidToken
+
+from users.models import CustomUser
+
+
+AUTH_HEAR_KEY = b'authorization'
```

Comment:
Suppose, the variable name `AUTH_HEAR_KEY` has a typo. Maybe, `AUTH_HEADER_KEY` sounds better.